TS 5.9 LibCheck error

### Vue version

3.6.0-beta.5

### Link to minimal reproduction

https://stackblitz.com/edit/vitejs-vite-cofjjwcc?file=README.md&view=editor

### Steps to reproduce

Typescript 5.9 with`skibLibCheck: false` and a vue file. Then call `vue-tsc`




### What is expected?

No typescript errors.

### What is actually happening?

`error TS2578: Unused '@ts-expect-error' directive.`

### System Info

```shell

```

### Any additional comments?

_No response_

--- a/packages/vue/jsx-runtime/index.d.ts
+++ b/packages/vue/jsx-runtime/index.d.ts
@@ -17,7 +17,7 @@ export namespace JSX {
   }
   export interface IntrinsicElements extends NativeElements {
     // allow arbitrary elements
-    // @ts-expect-error suppress ts:2374 = Duplicate string index signature.
+    // @ts-ignore suppress ts:2374 = Duplicate string index signature.
     [name: string]: any
   }
   export interface IntrinsicAttributes extends ReservedProps {}

